source/docs/migration/new-to-phoenix.rst

```diff
@@ -44,6 +44,12 @@ Free :doc:`High-Fidelity Simulation </docs/api-reference/simulation/simulation-i
 - Simulation closely follows the behavior of real hardware.
 - Write unit-tests for your robot code, and make sure the robot works before deploying.
 
+:ref:`Continuous Wrap Mode <docs/api-reference/device-specific/talonfx/closed-loop-requests:continuous mechanism wrap>`
```

Comment:
Should we also add this to the feature table?

Reply:
Yeah, we should